apps/etl/etl_tasks/usgs.py

```diff
@@ -0,0 +1,10 @@
+from celery import shared_task
+
+from apps.etl.extraction.sources.usgs.extract import (
+    import_hazard_data as import_usgs_data,
+)
+
+
+@shared_task
+def import_usgs_hazard_data(**kwargs):
+    import_usgs_data()
```

Comment:
lets add extraction func and transformation function in  a single func inside etl_task/usgs.py file and call the function in management command using delay().
```suggestion
        data = import_usgs_data()
        transform_usgs_event_data(data)
```

Reply:
Our concern is, In tasks file we just call the function for extraction and transformation.